Vagrantfile

```diff
@@ -111,7 +111,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if testing?
         app.vm.synced_folder ".", "/opt/app"
     else
-        app.vm.synced_folder ".", "/opt/app", :nfs => true, :mount_options => ['actimeo=2']
+        app.vm.synced_folder ".", "/opt/app", :nfs => true, :mount_options => [
+            ("nfsvers=3" if ENV.fetch("CAC_NFS_VERSION", 4).to_i == 3),
```

Comment:
Yikes, that's what i get for copy-pasting. I'll update to what you suggest.

Fix is from here: https://github.com/azavea/nih-wayfinding/pull/52